great_expectations/execution_engine/execution_engine.py

```diff
@@ -527,6 +458,208 @@ def resolve_data_reference(
             template_arguments=template_arguments,
         )
 
+    def _build_direct_and_bundled_metric_computation_configurationsd(
+        self,
+        metrics_to_resolve: Iterable[MetricConfiguration],
+        metrics: Optional[Dict[Tuple[str, str, str], MetricValue]] = None,
+        runtime_configuration: Optional[dict] = None,
+    ) -> Tuple[
+        List[MetricComputationConfiguration], List[MetricComputationConfiguration]
```

Comment:
It would be useful to make this a `NamedTuple` so you don't inadvertently swap these two lists.